test/Microsoft.AspNetCore.Razor.Language.Test/TestFiles/ParserTests/TagHelperParseTreeRewriterTest/DoesNotRewriteSpecialTagTagHelpers1.diag.txt

```diff
@@ -0,0 +1,5 @@
+(0,0): Error RZ3008: Tag helpers cannot target tag name '!--' because it contains a '!' character.
+(0,0): Error RZ3008: Tag helpers cannot target tag name '?xml' because it contains a '?' character.
+(0,0): Error RZ3008: Tag helpers cannot target tag name '![CDATA[' because it contains a '!' character.
+(0,0): Error RZ3008: Tag helpers cannot target tag name '![CDATA[' because it contains a '[' character.
+(0,0): Error RZ3008: Tag helpers cannot target tag name '!DOCTYPE' because it contains a '!' character.
```

Comment:
This is not a bug. These are the errors generated when the tag helper descriptors are found. Previously, `TagHelperParseTreeRewriter` was not responsible for combing these errors so test code never saw that. But I had to refactor some of that code and now the errors are [combined ](https://github.com/aspnet/Razor/blob/ajbaaska/taghelper-parser/src/Microsoft.AspNetCore.Razor.Language/Legacy/TagHelperParseTreeRewriter.cs#L33) in `TagHelperParseTreeRewriter`. Hence these errors show up.